src/index.html

```diff
@@ -1,13 +1,78 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>BOSE</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap"    rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a href="/" class="logo">
+              <img 
+                class="logo__image" 
+                src="images/logo.png" 
+                alt="BOSE"
+                >
+            </a>
+
+            <a 
+              href="Tel:+1 234 555-55-55" 
+              class="icon icon--phone header__call"
+            ></a>
+
+            <a href="#" class="icon icon--menu">
+            </a>
+          </div>
+          
+          <h1 class="header__title">
+            <div class="header__rectangles">
+              <div class="header__rectangles-item header__rectangles-item--1 "></div>
```

Comment:
don't add each line separately
you can select them all end export from Figma as 1 SVG file:
![image](https://user-images.githubusercontent.com/86923089/193451268-8f0f556d-5bd5-4e73-9499-120020ce4ee0.png)